### ISSUE TITLE
Unicode player paths not supported

### ISSUE THREAD

**Issue by Et0h** [Syncplay/syncplay#182]
It was reported today on IRC that Unicode characters in player paths can cause things to break, at least on Windows.

Example error from running mpv with the ğ chracter in the path:

```
Exception in thread GetPlayerIcon:
Traceback (most recent call last):
  File "threading.pyo", line 801, in __bootstrap_inner
  File "syncplay\ui\GuiConfiguration.pyo", line 71, in run
  File "syncplay\players\playerFactory.pyo", line 20, in getPlayerIconByPath
  File "syncplay\players\mpv.pyo", line 52, in isValidPlayerPath
  File "syncplay\players\mpv.pyo", line 68, in getExpandedPath
  File "ntpath.pyo", line 85, in join
UnicodeDecodeError: 'ascii' codec can't decode byte 0xf0 in position 9: ordinal not in range(128)
```
An easy work-around is not to use Unicode in player paths. However, if someone wants to code a fix for this issue then that would be helpful.

**Comment by albertosottile** [comment]
Unable to reproduce with mpv on macOS, it might be Windows-related.

**Comment by albertosottile** [comment]
Just an update: this bug is not present in the Python 3.x build that I am currently testing (tested on Windows 10 x64 on VLC and MPC-HC). I guess we will have to test this again before releasing it, so this comment should act as a reminder.

**Comment by Et0h** [comment]
Resolved by #191
